src/main/python/net/i2cat/cnsmo/app/vpn/configurator.py

```diff
@@ -121,22 +121,22 @@ def generate_ca_and_dh(self):
         command = "sh %s../gen_ca.sh" % self.key_dir
         subprocess.check_call(shlex.split(command), shell=False)
 
-    def generate_client_certs(self):
-        command = "sh %s../gen_client.sh" % self.key_dir
+    def generate_client_certs(self, client_id):
+        command = "sh %s../gen_client.sh -u %s" % (self.key_dir, client_id)
         subprocess.check_call(shlex.split(command))
 
     def generate_server_certs(self):
         command = "sh %s../gen_server.sh" % self.key_dir
         subprocess.check_call(shlex.split(command))
 
-    def get_client_cert(self):
-        f = open(self.key_dir + "client.crt")
+    def get_client_cert(self, client_id):
```

Comment:
Created a jira ticket to address this issue: http://jira.i2cat.net/browse/SEG-434